ui/js/directives/treeherder/clonejobs.js

```diff
@@ -279,6 +284,7 @@ treeherder.directive('thCloneJobs', [
 
             var jobList = platformGroup.find(".group-job-list");
             var countList = platformGroup.find(".group-count-list");
+            var typeSymbolCounts = _.countBy(jgObj.jobs, _getJobTypeSymbol);
```

Comment:
The docs are suboptimal on countBy (https://lodash.com/docs#countBy), but from reading between the lines, it also accepts a property name (passed as a string) as an alternative to a function - meaning we could use:
`_.countBy(jgObj.jobs, 'job_type_symbol');`
...and not need to define the function at all :-)